lib/validation/rules/ymd.ts

```diff
@@ -1,14 +1,18 @@
 import { helpers } from '@vuelidate/validators'
 import { type Ymd } from '../../support/Day'
+import { createRule } from '../Rule'
 import { ymd as baseYmd } from '../validators/ymd'
 
 type YmdType = 'y' | 'm' | 'd'
 
+export const message = {
+  en: 'The field is required.',
+  ja: 'この項目は必須です。'
+}
+
 export function ymd(required?: YmdType[], msg?: string) {
-  return helpers.withMessage(
-    () => msg ?? 'The date is invalid.',
-    (value: Ymd) => {
-      return !helpers.req(value) || baseYmd(value, required)
-    }
-  )
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang],
+    validation: (value: Ymd) => !helpers.req(value) || baseYmd(value, required)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70